src/epoch.rs

```diff
@@ -675,6 +704,33 @@ impl Epoch {
         self.as_gpst_duration().in_unit(Unit::Day)
     }
 
+    ///Returns the Duration since the UNIX epoch UTC midnight 01 Jan 1970.
+    pub fn as_unix_duration(self) -> Duration {
+        let cnt = self.get_num_leap_seconds();
+        // TAI = UNIX + leap_seconds + UNIX_OFFSET_UTC_SECONDS <=> UNIX = TAI - leap_seconds - UNIX_OFFSET_UTC_SECONDS
+        self.0 + i64::from(-cnt) * Unit::Second - UNIX_OFFSET_UTC_SECONDS * Unit::Second
+    }
+
+    /// Returns the duration since the UNIX epoch in the provided unit.
+    pub fn as_unix(self, unit: Unit) -> f64 {
+        self.as_unix_duration().in_unit(unit)
+    }
+
+    /// Returns the number seconds since the UNIX epoch defined 01 Jan 1970 midnight UTC.
+    pub fn as_unix_seconds(self) -> f64 {
+        self.as_unix(Unit::Second)
+    }
+
+    /// Returns the number miliseconds since the UNIX epoch defined 01 Jan 1970 midnight UTC.
+    pub fn as_unix_miliseconds(self) -> f64 {
```

Comment:
Milliseconds has two `l`s.
```suggestion
    pub fn as_unix_milliseconds(self) -> f64 {
```